src/Controller/RecurlySubscriptionListController.php

```diff
@@ -23,6 +25,32 @@
 class RecurlySubscriptionListController extends ControllerBase {
 
   /**
+   * The formatting service.
+   *
+   * @var \Drupal\recurly\RecurlyFormatManager
+   */
+  protected $formatter;
```

Comment:
My thought was if there was some kind of core or contrib formatter service that we may want/need to include in the future it may make things more clear. I defer to you.